src/api/authentication.coffee

```diff
@@ -3,6 +3,23 @@ crypto = require 'crypto'
 logger = require 'winston'
 config = require "../config/config"
 config.api = config.get('api')
+config.auditing = config.get('auditing')
+atna = require 'atna-audit'
+auditing = require '../auditing'
+os = require 'os'
+himSourceID = config.auditing.auditEvents.auditSourceID
+
+# will NOT audit any successful logins on the following paths (specified as regex patterns)
+# only 'noisy' endpoints should be included, such as heartbeats or endpoints that get polled
+auditingExemptPaths = [
+  /\/transactions/
```

Comment:
Yup it does - yeah good idea. We can add a check to only exempt if on a simple view.